src/test/java/org/broadinstitute/hellbender/engine/dataflow/RefAPISourceUnitTest.java

```diff
@@ -112,6 +113,28 @@ public void testReferenceSourceQueryWithNullInterval() {
     }
 
 
+    @Test(groups = "cloud")
+    public void testReferenceSourceMultiPageQuery() {
+        final int mio = 1_000_000;
```

Comment:
ok, except rather than making the static visible (which could be problematic because changing it in one part of the code would change the behavior of every other part of the code) I turned it into an optional argument.